kaolin/visualize/timelapse.py

```diff
@@ -14,6 +14,10 @@
 
 logger = logging.getLogger(__name__)
 
+__all__ = [
```

Comment:
What is this and why is this needed? 

Please update the `doc/README.md` to reflect any tricks on excluded files and these `__all__` tricks. The current README is very nice, but is missing these details.

Reply:
This `__all__` is not only for documentation.
This affect the behavior of `from mymodule import *` that I added in https://github.com/Caenorst/kaolin/blob/cfujitsang/auto_doc/kaolin/visualize/__init__.py#L1

If you don't make an `__all__` then `from mymodule *` will return **any** variable instantiated in `mymodule`, so for instance all the imported modules: https://github.com/Caenorst/kaolin/blob/cfujitsang/auto_doc/kaolin/visualize/timelapse.py#L1-L15.

I believe this make contribution easier, to add a new feature in the file, as all the modifications will be contained in the same file (here `kaolin/visualize/timelapse.py`)

Reply:
updated docs/README.md